### DISCUSSION TITLE
after set basePath, _next/static/chunks/pages return 404

### DISCUSSION THREAD

**Discussion by erlangparasu** [vercel/next.js#36541] [Help]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

Operating System:
      Platform: linux
      Arch: x64
      Version: #1 SMP Thu Apr 22 16:18:59 MSK 2021
    Binaries:
      Node: 16.14.2
      npm: 8.7.0
      Yarn: 1.22.18
      pnpm: N/A
    Relevant packages:
      next: 12.1.5
      react: 17.0.2
      react-dom: 17.0.2

### What browser are you using? (if relevant)

Firefox 99.0.1 (64-bit)

### How are you deploying your application? (if relevant)

next build

### Describe the Bug

after set basePath, _next/static/chunks/pages return 404

### Expected Behavior

after set basePath, _next/static/chunks/pages return 200

### To Reproduce

set custom basePath in next.config.js

**Comment by stanleywanone** [comment]
I have the same issue, and still cannot find the solution
